portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,45 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+/**
+* Creating a servlet that stores comments as a JSON list and use JavaScript that builds UI from
+* that data. 
+**/
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+
+
+
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Sean Tisdale!</h1>");
+   // Converts message to JSON string
+   String message = messageGson(messages());
+    response.setContentType("application/json;");
+   response.getWriter().println(message);
+
+  }
+    // Will have messages that are presented on the screen
+    public String messages() {
```

Comment:
1. Indent -2
2. Add blank line before this method

Reply:
when you say add a blink do you mean between the comment and the method?

Reply:
Between the comment and the *previous* method.

Reply:
solved